src/tavi/frontend/views/main_view.py

```diff
@@ -83,24 +80,30 @@ def __init__(self, parent=None):
         self.setWindowTitle(f"TAVI - {__version__}")
         self.main_window = MainWindow(self)
         self.setCentralWidget(self.main_window)
+        self._force_closing = False
 
     def install_menu_bar(self, menu_bar):
         """Called by MainPresenter to attach the menu bar."""
         self.setMenuBar(menu_bar)
 
+
     def closeEvent(self, event):
```

Comment:
Should this not just "callback" to the presenter?  Could we handle the Tavi class exit callback like we are with FileMenu with a `setup_callback_exit(self, callback: None) -> None:` method?  This would make things consistent to me.